core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/get/TransportGetTaskAction.java

```diff
@@ -203,7 +204,7 @@ void getFinishedTaskFromIndex(Task thisTask, GetTaskRequest request, ActionListe
                 request.getTaskId().toString());
         get.setParentTask(clusterService.localNode().getId(), thisTask.getId());
         client.get(get, new ActionListener<GetResponse>() {
-            @Override
+            @Override @SwallowsExceptions(reason = "?")
```

Comment:
I dont think we should try to fix any of the problems here? it would be great to simply document them all with the annotation and try to fix them separately. there are still 700 un-annotated violations in es core alone. I tried to start annotating them, its too exhausting.